pkg/runtime/wfengine/wfengine.go

```diff
@@ -46,37 +42,50 @@ type WorkflowEngine struct {
 	actorRuntime   actors.Actors
 	startMutex     sync.Mutex
 	disconnectChan chan any
+	config         *WFConfig
 }
 
 var (
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID             string
+	WorkflowActorType string
+	ActivityActorType string
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	config.WorkflowActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".workflow"
+	config.ActivityActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".activity"
```

Comment:
have `.` as a separator and const and have workflow and activity as consts as well.

Reply:
Please check now.